atmo/jobs/forms.py

```diff
@@ -26,9 +26,20 @@ class BaseSparkJobForm(FormControlFormMixin, CachedFileModelFormMixin,
             'class': 'identifier-taken-check',
             'data-identifier-taken-check-url': reverse_lazy('jobs-identifier-taken'),
         }),
-        help_text='A brief description of the scheduled Spark job\'s purpose, '
+        help_text='A slug-like string to identify your Spark job, '
```

Comment:
"slug-like" is a bit technical and not likely to be understood by the ATMO users, can we find a middleground here?